docs/architecture.rst

```diff
@@ -149,6 +152,7 @@ as follows:
       ConfigMaps map[string]ObjectReference `json:"configMaps"`
       Secrets map[string]ObjectReference    `json:"secrets"`
       Profile *ObjectReference              `json:"profile"`
+      Options map[string]string             `json:"options"`
```

Comment:
[minor] Please check formatting here. Looks like _italics_ is enabled.

Reply:
@SupriyaKasten I think this is because of `*ObjectReference`. * enabled italics.

Reply:
Sure!